connector/src/it/scala/com/datastax/spark/connector/writer/TableWriterSpec.scala

```diff
@@ -771,7 +771,7 @@ class TableWriterSpec extends SparkCassandraITFlatSpecBase with DefaultCluster {
     verifyKeyValueTable("key_value")
   }
 
-  it should "be able to append and prepend elements to a C* list" in {
+  it should "be able to.append and.prepend elements to a C* list" in {
```

Comment:
Could you remove dots from test name here and others below?

Reply:
Fixed in [[SPARKC-686] Remove accidental dots in spec names](https://github.com/datastax/spark-cassandra-connector/pull/1361/commits/c6d84c6b395f343903dc7d61aec3eb99da966885)